app/views/gallery/index.html.erb

```diff
@@ -1,6 +1,13 @@
 <%= title(t('gallery.title')) %>
 <div class="headline headline-md">
-  <h1><%= t('gallery.title') %></h1>
+  <h1>
+    <%= t('gallery.title') %> -
+    <% if @year.present? %>
```

Comment:
It's ok to have two variables, I just think the names are confusing.

Maybe this_year and other_years? Or something just to clarify what they mean.